internal/handlers/pg/pgdb/collections.go

```diff
@@ -35,7 +35,8 @@ import (
 
 // validateCollectionNameRe validates collection names.
 // Empty collection name, names with `$` and `\x00` are not allowed.
-var validateCollectionNameRe = regexp.MustCompile("^[^$\x00]{1,235}$")
+// Collection names that start with `.` are also not allowed.
+var validateCollectionNameRe = regexp.MustCompile("^[^.$\x00][^$\x00]{0,234}$")
```

Comment:
The dot part of this regex is checked in if statement so it's not necessary here, although I'd suggest to leave it here, to make it understandable for us what's the full collection name regex.